packages/remirror__extension-node-formatting/src/node-formatting-extension.ts

```diff
@@ -51,7 +53,7 @@ import {
       '180px',
       '200px',
     ],
-    excludeNodes: [],
+    excludeNodes: ['bulletList', 'listItem'],
```

Comment:
do we need to add `orderedList`, `taskList` and `taskListItem` here?

It is possible to use the group name here? i.e. `ListContainerNode` and `ListItemNode`?

Reply:
Added `orderedList`, `taskList` and `taskListItem`.

We have to edit `@remirorr/core` to support group name here. I kind of don't want to touch `core` unless very necessary.